manifests/mysql_connector.pp

```diff
@@ -0,0 +1,27 @@
+class confluence::mysql_connector (
+  String $connector_version  = $confluence::mysql_connector_version,
```

Comment:
Are these namespace variables on the right hand side available? This does not inherit the $confluence class. This could be why the file locations don’t match. You can see what files are included with an `it { pp catalogue.resources }` test block.

Reply:
will try that thanks !!

Reply:
variable was fine, Just got a wrong path /opt/ which causes troubles and then I had a mistake on confluence version. thanks !